### DISCUSSION TITLE
Issue with Basset on local

### DISCUSSION THREAD

**Discussion by mariusconstantin2503** [Laravel-Backpack/community-forum#812] [General]
I am trying to install a project based on Backpack which is using the Baset package. When I try to access the admin part of the project I receive a weird error:
![image](https://github.com/Laravel-Backpack/community-forum/assets/4282568/cc665af8-e367-4277-bde4-e6e9e5d4ceff)
https://flareapp.io/share/LPl9R4B7
Any idea what it can be or how can I solve it?

**Comment by pxpm** [comment]
Hey @mariusconstantin2503 

What's the output of `php artisan basset:check` ? 

What's in your `.env` APP_URL value ? 

Cheers

**Reply by mariusconstantin2503** [reply]
I think I fixed it. The filesystem was set to S3. When I changed it to public it worked.
